apps/tables/package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@mswertz/emx2-styleguide": "*",
```

Comment:
huh, it is surprising it worked at all???

Reply:
side effect of the shared node modules, i think , bit sketchy